src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/utilities/HeapHprofBinWriter.java

```diff
@@ -1522,6 +1574,8 @@ private final void writeInteger(int value) {
         // Segment support.
         private boolean segmentMode;
         private boolean allowSegmented;
+        // Writes data directly to underlying stream, don't use internal buffer.
```

Comment:
It should be "Write data". I noticed you recently changed it to "Writes", but I'm not sure why. Also, make this two sentences rather than using a comma.